<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Mohammed Omar Elsiddieg Abdallah" />


<title>MutationnSignature Analysis</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GBM_Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">MutationnSignature Analysis</h1>
<h4 class="author"><em>Mohammed Omar Elsiddieg Abdallah</em></h4>
<h4 class="date"><em>9/20/2018</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-09-28</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<details>
<p><summary> <strong style="color:red;">✖</strong> <strong>R Markdown file:</strong> uncommitted changes </summary> The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180928)</code> </summary></p>
<p>The command <code>set.seed(20180928)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<details>
<p><summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 72fc62c </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/GBM_Analysis_cache/

Untracked files:
    Untracked:  24_changePoints.tsv
    Untracked:  Brn_sig.maf
    Untracked:  Mutational_Signature_files/
    Untracked:  analysis/GBM_Analysis.Rmd
    Untracked:  analysis/Mutational_Signature.Rmd
    Untracked:  docs/figure/

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</details>
</li>
</ul>
<hr />
<div id="reading-reference-genome" class="section level2">
<h2>Reading reference genome</h2>
<pre class="r"><code>ref_genome &lt;- &quot;BSgenome.Hsapiens.UCSC.hg19&quot;
library(ref_genome, character.only = TRUE)</code></pre>
<pre><code>Loading required package: BSgenome</code></pre>
<pre><code>Loading required package: rtracklayer</code></pre>
<pre class="r"><code>faf &lt;- FaFile(&#39;/home/sieny/snpEff/Human_Ref/hg19.fa&#39;)</code></pre>
</div>
<div id="reading-vcf-files" class="section level2">
<h2>Reading VCF files</h2>
<pre class="r"><code>setwd(&#39;/home/sieny/Projects/GBM/New_Threshold/&#39;)
files &lt;- list.files(pattern = &#39;passOnly.vcf$&#39;)
# Exclude normal sample
files &lt;- files[-19]
fnames &lt;- str_split(files,pattern = &quot;\\.&quot;) %&gt;% sapply( function(x)x[1])
head(fnames)</code></pre>
<pre><code>[1] &quot;tumor_1_snvs_indels_filtered_passOnly&quot; 
[2] &quot;tumor_10_snvs_indels_filtered_passOnly&quot;
[3] &quot;tumor_11_snvs_indels_filtered_passOnly&quot;
[4] &quot;tumor_12_snvs_indels_filtered_passOnly&quot;
[5] &quot;tumor_13_snvs_indels_filtered_passOnly&quot;
[6] &quot;tumor_14_snvs_indels_filtered_passOnly&quot;</code></pre>
<pre class="r"><code>snames &lt;- str_extract(fnames,&#39;[a-z]?\\d+&#39;)</code></pre>
</div>
<div id="get-mutations-from-vcf" class="section level2">
<h2>Get Mutations from VCF</h2>
<pre class="r"><code>muts = mutations_from_vcf(vcfs[[1]])
head(muts,12)</code></pre>
<pre><code> [1] &quot;G&gt;A&quot; &quot;T&gt;C&quot; &quot;C&gt;T&quot; &quot;A&gt;G&quot; &quot;G&gt;A&quot; &quot;A&gt;G&quot; &quot;C&gt;T&quot; &quot;A&gt;G&quot; &quot;G&gt;A&quot; &quot;G&gt;A&quot; &quot;A&gt;G&quot;
[12] &quot;C&gt;A&quot;</code></pre>
</div>
<div id="mutation-context" class="section level2">
<h2>Mutation Context</h2>
<pre class="r"><code>context = mut_context(vcfs[[1]], ref_genome)
head(context, 12)</code></pre>
<pre><code> chr1  chr1  chr1  chr1  chr1  chr1  chr1  chr1  chr1  chr1  chr1  chr1 
&quot;CGG&quot; &quot;CTG&quot; &quot;TCA&quot; &quot;CAG&quot; &quot;GGG&quot; &quot;AAT&quot; &quot;GCG&quot; &quot;GAT&quot; &quot;CGG&quot; &quot;CGC&quot; &quot;CAC&quot; &quot;GCC&quot; </code></pre>
</div>
<div id="mutation-spectrum" class="section level2">
<h2>Mutation Spectrum</h2>
<pre class="r"><code> type_occurrences &lt;- mut_type_occurrences(vcfs, ref_genome)
 type_occurrences</code></pre>
<pre><code>    C&gt;A  C&gt;G  C&gt;T T&gt;A  T&gt;C T&gt;G C&gt;T at CpG C&gt;T other
1   515  298 1200 227  912 239        385       815
10  550  418 1806 268 1382 369        634      1172
11   95   49  277  31  183  42        101       176
12  285  210  809 116  564 154        313       496
13  282  180 1199 649  905  64        270       929
14  572  362 1446 198 1082 252        509       937
16  278  202  887 142  687 158        292       595
17  363  100  421  62  274  56        171       250
18  244   58  217  33  146  37         78       139
2  1477 1115 4300 617 3240 776       1654      2646
20 2749  611 2350 361 1732 462        871      1479
21  389  156  608  88  443 105        222       386
22  498  172  639 113  478 128        220       419
23 3856 1065 4176 623 3132 795       1618      2558
24 2734  327 3436 300 1521 613       1411      2025
25  237  181  648  98  495 122        220       428
27  545  369 1564 232 1111 284        608       956
28   95   36  194  26  120  32         74       120
3     0    0    1   0    0   0          1         0
30 1554  996 3874 554 2934 720       1512      2362
31  873  478 1877 295 1433 378        649      1228
32  556  332 1391 220 1043 241        458       933
4   316  233 1029 143  736 198        383       646
5   721  597 2345 388 1838 464        917      1428
6    10   11   47   5   28   6         24        23
7   242  171  716 141  509 105        279       437
8   233  153  589  93  460 118        212       377
9   459  332 1264 228 1039 239        438       826
c6  539  417 1730 266 1339 304        656      1074</code></pre>
<pre class="r"><code> plot_spectrum(type_occurrences)</code></pre>
<p><img src="figure/Mutational_Signature.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="profile" class="section level2">
<h2>96 profile</h2>
<pre class="r"><code>mut_mat &lt;- mut_matrix(vcf_list = vcfs, ref_genome = ref_genome)
plot_96_profile(mut_mat[,c(1,7)])</code></pre>
<p><img src="figure/Mutational_Signature.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /> ## De Novo Signature</p>
<pre class="r"><code>mut_mat &lt;- mut_mat + 0.0001
library(&quot;NMF&quot;)
#estimate &lt;- nmf(mut_mat, rank=2:5, method=&quot;brunet&quot;, nrun=10, seed=123456)</code></pre>
</div>
<div id="cosmic-signature" class="section level2">
<h2>Cosmic Signature</h2>
<pre class="r"><code> sp_url &lt;- paste(&quot;http://cancer.sanger.ac.uk/cancergenome/assets/&quot;,
 &quot;signatures_probabilities.txt&quot;, sep = &quot;&quot;)
cancer_signatures = readr::read_delim(sp_url,delim = &quot;\t&quot;)</code></pre>
<pre><code>Warning: Missing column names filled in: &#39;X34&#39; [34], &#39;X35&#39; [35],
&#39;X36&#39; [36], &#39;X37&#39; [37], &#39;X38&#39; [38], &#39;X39&#39; [39], &#39;X40&#39; [40]</code></pre>
<pre><code>Parsed with column specification:
cols(
  .default = col_double(),
  `Substitution Type` = col_character(),
  Trinucleotide = col_character(),
  `Somatic Mutation Type` = col_character(),
  X34 = col_character(),
  X35 = col_character(),
  X36 = col_character(),
  X37 = col_character(),
  X38 = col_character(),
  X39 = col_character(),
  X40 = col_character()
)</code></pre>
<pre><code>See spec(...) for full column specifications.</code></pre>
<pre class="r"><code>new_order = match(row.names(mut_mat), cancer_signatures$`Somatic Mutation Type`)</code></pre>
<pre class="r"><code>cancer_signatures = cancer_signatures[as.vector(new_order),]
cancer_signatures &lt;- as.data.frame(cancer_signatures)
row.names(cancer_signatures) = cancer_signatures$`Somatic Mutation Type`</code></pre>
<pre class="r"><code>cancer_signatures = as.matrix(cancer_signatures[,4:33])</code></pre>
<pre class="r"><code>sig_hclust = cluster_signatures(cancer_signatures)
col_order = colnames(cancer_signatures)[sig_hclust$order]
plot_96_profile(cancer_signatures[,1:4], condensed = TRUE, ymax = 0.3)</code></pre>
<p><img src="figure/Mutational_Signature.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="cosmic-similarity" class="section level2">
<h2>Cosmic Similarity</h2>
<pre class="r"><code> cos_sim(mut_mat[,1], cancer_signatures[,1])</code></pre>
<pre><code>[1] 0.6784088</code></pre>
<pre class="r"><code> cos_sim_samples_signatures = cos_sim_matrix(mut_mat, cancer_signatures)
 plot_cosine_heatmap(cos_sim_samples_signatures,
 col_order = col_order,
 cluster_rows = TRUE)</code></pre>
<p><img src="figure/Mutational_Signature.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.0 (2018-04-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.5 LTS

Matrix products: default
BLAS: /opt/microsoft/ropen/3.5.0/lib64/R/lib/libRblas.so
LAPACK: /opt/microsoft/ropen/3.5.0/lib64/R/lib/libRlapack.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] BSgenome.Hsapiens.UCSC.hg19_1.4.0 BSgenome_1.48.0                  
 [3] rtracklayer_1.40.6                stringr_1.3.1                    
 [5] data.table_1.11.4                 SomaticSignatures_2.16.0         
 [7] VariantAnnotation_1.26.1          Rsamtools_1.32.3                 
 [9] Biostrings_2.48.0                 XVector_0.20.0                   
[11] SummarizedExperiment_1.10.1       DelayedArray_0.6.6               
[13] BiocParallel_1.14.2               matrixStats_0.54.0               
[15] MutationalPatterns_1.6.1          NMF_0.21.0                       
[17] Biobase_2.40.0                    cluster_2.0.7-1                  
[19] rngtools_1.3.1                    pkgmaker_0.27                    
[21] registry_0.5                      GenomicRanges_1.32.6             
[23] GenomeInfoDb_1.16.0               IRanges_2.14.11                  
[25] S4Vectors_0.18.3                  BiocGenerics_0.26.0              

loaded via a namespace (and not attached):
  [1] colorspace_1.3-2         rprojroot_1.3-2         
  [3] biovizBase_1.28.0        htmlTable_1.12          
  [5] base64enc_0.1-3          ggdendro_0.1-20         
  [7] dichromat_2.0-0          proxy_0.4-22            
  [9] rstudioapi_0.7           bit64_0.9-7             
 [11] AnnotationDbi_1.42.1     codetools_0.2-15        
 [13] splines_3.5.0            R.methodsS3_1.7.1       
 [15] ggbio_1.28.0             doParallel_1.0.13       
 [17] knitr_1.20               Formula_1.2-3           
 [19] workflowr_1.0.1          gridBase_0.4-7          
 [21] R.oo_1.22.0              graph_1.58.0            
 [23] readr_1.1.1              compiler_3.5.0          
 [25] httr_1.3.1               backports_1.1.2         
 [27] assertthat_0.2.0         Matrix_1.2-14           
 [29] lazyeval_0.2.1           acepack_1.4.1           
 [31] htmltools_0.3.6          prettyunits_1.0.2       
 [33] tools_3.5.0              bindrcpp_0.2.2          
 [35] gtable_0.2.0             glue_1.3.0              
 [37] GenomeInfoDbData_1.1.0   reshape2_1.4.3          
 [39] dplyr_0.7.6              Rcpp_0.12.18            
 [41] iterators_1.0.10         ensembldb_2.4.1         
 [43] XML_3.98-1.16            zlibbioc_1.26.0         
 [45] MASS_7.3-49              scales_1.0.0            
 [47] BiocInstaller_1.30.0     pcaMethods_1.72.0       
 [49] ProtGenerics_1.12.0      hms_0.4.2               
 [51] RBGL_1.56.0              AnnotationFilter_1.4.0  
 [53] RColorBrewer_1.1-2       curl_3.2                
 [55] yaml_2.1.19              memoise_1.1.0           
 [57] gridExtra_2.3            ggplot2_3.0.0           
 [59] biomaRt_2.36.1           rpart_4.1-13            
 [61] reshape_0.8.7            latticeExtra_0.6-28     
 [63] stringi_1.2.4            RSQLite_2.1.1           
 [65] foreach_1.5.0            checkmate_1.8.5         
 [67] GenomicFeatures_1.32.2   bibtex_0.4.2            
 [69] rlang_0.2.2              pkgconfig_2.0.2         
 [71] bitops_1.0-6             pracma_2.1.5            
 [73] evaluate_0.10.1          lattice_0.20-35         
 [75] purrr_0.2.5              bindr_0.1.1             
 [77] labeling_0.3             GenomicAlignments_1.16.0
 [79] htmlwidgets_1.2          cowplot_0.9.3           
 [81] bit_1.1-14               tidyselect_0.2.4        
 [83] GGally_1.4.0             plyr_1.8.4              
 [85] magrittr_1.5             R6_2.2.2                
 [87] Hmisc_4.1-1              DBI_1.0.0               
 [89] pillar_1.3.0             whisker_0.3-2           
 [91] foreign_0.8-70           withr_2.1.2             
 [93] survival_2.41-3          RCurl_1.95-4.11         
 [95] nnet_7.3-12              tibble_1.4.2            
 [97] crayon_1.3.4             OrganismDbi_1.22.0      
 [99] rmarkdown_1.9            progress_1.2.0          
[101] grid_3.5.0               blob_1.1.1              
[103] git2r_0.21.0             digest_0.6.17           
[105] xtable_1.8-3             R.utils_2.6.0           
[107] munsell_0.5.0           </code></pre>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.0.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
