---
title: "GBM Analysis Report"
author: "Mohammed Omar Elsiddieg Abdallah"
date: "9/12/2018"
output:
  html_document: default
  pdf_document: default
---

```{r , include=FALSE}
knitr::opts_chunk$set(echo = TRUE, message = FALSE)
```


## Read in the Data
```{r setup, message=FALSE}
library(maftools)
library(data.table)
maf <- fread('/home/sieny/Gemini_Data/Brain/mskcc-vcf2maf-decbf60/Annoted_Mutect_brain6_final.maf')
clin <- fread('/home/sieny/Gemini_Data/Brain/cData.csv')
brn <- read.maf(maf,clin)
```
## Explore
```{r explore}
getSampleSummary(brn)
getGeneSummary(brn)
getFields(brn)
```
## Maf summary

```{r mafsummary}
plotmafSummary(maf = brn, rmOutlier = TRUE, addStat = 'median', dashboard = TRUE, titvRaw = FALSE)
```

## Oncoplot

```{r oncoplot}
oncoplot(maf = brn, top = 20,clinicalFeatures = 'Classification',
         sortByAnnotation = TRUE,annotationColor = list(Classification=c(Low_Grade=c("#FFE4C4"),High_Grade=c("#8B4513"))))
```
<!-- Query Genes -->
```{r new_oncoplot}
genes <- c('TP53','ATRX','PGBM','NF2','BRAF','mTOR','TTN','IDH1','FRMPD4',
           'PKHD1L1','SPATA')
oncoplot(maf = brn, genes =genes)


```



## TiTv summary

```{r titv}
brn.titv = titv(maf = brn, plot = FALSE, useSyn = TRUE)
plotTiTv(res = brn.titv)
```
## Lolipop plots
```{r lolipop}
lollipopPlot(maf = brn, gene = 'IGFN1', AACol = 'HGVSp', showMutationRate = TRUE)
```
 
```{r lolipop2}
lollipopPlot(maf = brn, gene = 'MEGF6', AACol = 'HGVSp', showMutationRate = TRUE)
```

##  Rainfall plot

```{r rainfall}
rainfallPlot(maf = brn, detectChangePoints = TRUE, fontSize = 12, pointSize = 0.6)
```

##  Somatic Interaction

```{r somatic_interactions}
somaticInteractions(maf = brn, top = 25, pvalue = c(0.05, 0.1))

```
## Oncodrive

```{r oncodrive ,FALSE, message=FALSE, warning=FALSE,results='hide', cache=TRUE}
brn.drivers = oncodrive(maf = brn, AACol = 'HGVSp', minMut = 5, pvalMethod = 'zscore')
```
 
```{r oncodrive_onco}

plotOncodrive(res = brn.drivers, fdrCutOff = 0.1, useFraction = TRUE)
```

## PFAM domains

```{r pfam}
brn.pfam = pfamDomains(maf = brn, AACol = 'HGVSp', top = 10)
```

```{r tnm,include=FALSE}
brn.tnm = trinucleotideMatrix(maf = brn, 
                              ref_genome = '/home/sieny/snpEff/Human_Ref/hg19.fa', 
                              prefix="chr",add = TRUE, ignoreChr = 'chr23', useSyn = TRUE)
## plotApobecDiff(tnm = brn.tnm, maf = brn)
```

## Singnature analysis

```{r signature_analysis}
require('NMF')
brn.sign = extractSignatures(mat = brn.tnm, nTry = 6, plotBestFitRes = FALSE)
```

## Visualizing Signatures

```{r plot_signature}
plotSignatures(brn.sign, title_size = 0.8)
```

```{r signature_enrich, include=FALSE,eval=FALSE}
brn.se = signatureEnrichment(maf = brn, sig_res = brn.sign,minMut = 20)

```

## Clinical Enrichment
```{r clinical_enrich}
brn.ce = clinicalEnrichment(maf = brn, clinicalFeature = 'Classification')
plotEnrichmentResults(enrich_res = brn.ce, pVal = 0.1)


```

